datumaro/components/extractor.py

```diff
@@ -660,6 +731,10 @@ def __call__(self, path, **extra_params):
         if len(sources) == 0:
             raise Exception("Failed to find dataset at '%s'" % path)
 
+        for i, source in enumerate(sources):
```

Comment:
Do this in the importer class of the corresponding class.

Reply:
Resolved